crypto/fipsmodule/rand/asm/rdrand-x86_64.pl

```diff
@@ -69,15 +76,19 @@
 	movq \$8, $tmp1
 .Lloop:
 	rdrand $tmp2
-	jnc .Lerr
+	jnc .Lerr_multiple
```

Comment:
What's the reason to change the label? It seems acceptable to me to leave it as `.Lerr`.

Reply:
Label collision.